src/rez/utils/platform_.py

```diff
@@ -555,7 +557,8 @@ def _difftool(self):
 
 
 # singleton
-platform_ = None
+# FIXME: is is valid for platform_ to be None?
```

Comment:
Technically yes, say on solaris, freebsd, etc.

Reply:
allowing it to be None causes tons of mypy errors because there are lots of places where `platform_` is used without checking if it's None.

Reply:
And to be perfectly clear, these errors indicate that there are many places with the rez code that will fail at runtime if platform_ is None.

Reply:
Sure, I was just answering the question. It will fail if it's None. We could fix this by simply adding an else clause and raise a exception with a clear error message.